nncf/torch/extensions/__init__.py

```diff
@@ -98,9 +107,13 @@ def get(self, fn_name: str) -> Callable:
 
             with extension_is_loading_info_log(self._loader.name()):
                 try:
-                    pool = ThreadPool(processes=1)
-                    async_result = pool.apply_async(self._loader.load)
-                    self._loaded_namespace = async_result.get(timeout=timeout)
+                    with warnings.catch_warnings():
+                        warnings.filterwarnings("ignore", message="TORCH_CUDA_ARCH_LIST is not set")
+                        set_cuda_arch_list()
+
+                        pool = ThreadPool(processes=1)
+                        async_result = pool.apply_async(self._loader.load)
+                        self._loaded_namespace = async_result.get(timeout=timeout)
```

Comment:
I suggest does not set `TORCH_CUDA_ARCH_LIST` environment variable and just skip the warning.
```suggestion
                    with warnings.catch_warnings():
                        warnings.filterwarnings("ignore", message="TORCH_CUDA_ARCH_LIST is not set")
                        pool = ThreadPool(processes=1)
                        async_result = pool.apply_async(self._loader.load)
                        self._loaded_namespace = async_result.get(timeout=timeout)
```